autowarefoundation/autoware
# [MERGE AFTER #2072] Fix/refactoring autoware msgs

## Status
DEVELOPMENT

## Description
As part of `autoware_msgs` refactoring, in this PR we complete refactoring by removing the old `autoware_msgs` package plus some unused message types.
This was described at https://github.com/CPFL/Autoware/blob/feature/autoware_msgs_refactoring/ros/concerning-autoware-msgs.md, and further discussed at https://github.com/CPFL/Autoware/issues/1428.

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
feature/autoware_msgs_refactoring | [2066](https://github.com/CPFL/Autoware/pull/2066)
fix/refactoring_sync_msgs | [2067](https://github.com/CPFL/Autoware/pull/2067)
fix/refactoring_sensing_msgs | [2068](https://github.com/CPFL/Autoware/pull/2068)
fix/refactoring_localization_msgs | [2069](https://github.com/CPFL/Autoware/pull/2069)
fix/refactoring_planning_msgs | [2070](https://github.com/CPFL/Autoware/pull/2070)
fix/refactoring_socket_msgs | [2071](https://github.com/CPFL/Autoware/pull/2071)
fix/refactoring_detection_msgs | [2072](https://github.com/CPFL/Autoware/pull/2072)

## Todos


## Steps to Test or Reproduce
1. Get a fresh copy of Autoware `develop` branch, ex. `git clone https://github.com/CPFL/Autoware.git -b develop --recurse-submodules [AUTOWARE_PATH]`.
2. Make sure the original builds correctly: run `./catkin_make_release` (follow the steps in [How to build](https://github.com/CPFL/Autoware/wiki/Source-Build) ).
3. Switch to `fix/refactoring_sync_msgs` branch, ex. `git checkout fix/refactoring_sync_msgs` and pull for changes `git pull`.
4. Switch to `fix/refactoring_sensing_msgs` branch, ex. `git checkout fix/refactoring_sensing_msgs` and pull for changes `git pull`.
5. Switch to `fix/refactoring_localization_msgs` branch, ex. `git checkout fix/refactoring_localization_msgs` and pull for changes `git pull`.
6. Switch to `fix/refactoring_planning_msgs` branch, ex. `git checkout fix/refactoring_planning_msgs` and pull for changes `git pull`.
7. Switch to `fix/refactoring_socket_msgs` branch, ex. `git checkout fix/refactoring_socket_msgs` and pull for changes `git pull`.
8. Switch to `fix/refactoring_detection_msgs` branch, ex. `git checkout fix/refactoring_detection_msgs` and pull for changes `git pull`.
9. Switch to `fix/refactoring_autoware_msgs` branch, ex. `git checkout fix/refactoring_autoware_msgs` and pull for changes `git pull`.
10. Make sure refactored version builds correctly: run `./catkin_make_release`.
